### PR TITLE
feature: Support additional ServiceAccounts beyond the default account in each namespace

### PR DESCRIPTION
**Pull request by developer-guy**
## Description
Support additional ServiceAccounts beyond the default account in each namespace

Checklist:

- [ ] Fixes issue: # 
- [ ] I waited for approval before creating this PR

## How Has This Been Tested?
```sh
$ eval $(minikube docker-env)
$ make docker-build
$ make install deploy
$ arkade install ingress-nginx
$ arkade install cert-manager

$ kubectl create secret docker-registry registry-creds-secret \
  --namespace kube-system \
  --docker-username=$USERNAME \
  --docker-password=$PW \
  --docker-email=$EMAIL

$ kubectl create serviceaccount demo-sa 

$  echo -n "apiVersion: ops.alexellis.io/v1
kind: ClusterPullSecret
metadata:
  name: dockerhub
spec:
  secretRef:
    name: registry-creds-secret
    namespace: kube-system" | k apply -f -

$ kubectl get serviceaccounts default -oyaml 
$ kubectl get serviceaccounts demo-sa -oyaml
```

[
Screen Shot 2020-11-04 at 00 53 49
](url)
## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/inlets/inlets/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,13 +20,13 @@ COPY controllers/ controllers/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
-  GO111MODULE=on go build -ldflags="-w -s" -a -o /usr/bin/controller
```
Comment (alexellis) [inline Dockerfile]: My mistake. This should be `controller` everywhere. Can you update it?

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Todo:
 Apply the YAML for the manifest.
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/alexellis/registry-creds/master/mainfest.yaml
+kubectl apply -f https://raw.githubusercontent.com/alexellis/registry-creds/master/manifest.yaml
```
Comment (alexellis) [inline README.md:70]: Good catch

**Comment by alexellis** [comment]
@developer-guy please can you detail the testing that you did?

I saw that the "How has this been tested" section was removed from the PR template.

**Comment by alexellis** [comment]
Please add missing fields from https://raw.githubusercontent.com/alexellis/registry-creds/master/.github/pull_request_template.md

File: Dockerfile (hunk 1 above)
Comment (developer-guy) [reply Dockerfile]: Of course, done.

File: README.md (hunk 2 above)
Comment (developer-guy) [reply README.md:70]: Thank you.

**Pull request merged**